### DISCUSSION TITLE
Absorption Incomplete

### DISCUSSION THREAD

**Discussion by harrythmia** [Open-Systems-Pharmacology/Forum#963] [Questions & Problems] [answered]
Hello all, 

I have been working on RSV metabolism model for Hispanic population with CYP2C9 variants. However, the concentration-time curve that I got is showing me an infinite increase, instead of the first-order curve. 

All my parameters are predefined, I just added CYP2C9 in the "Compound" tab, under metabolizing enzyme. It also shows me error, saying that my absorption is incomplete.

Can anyone please help a newbie out? Thanks a lot!

Yours,
Harry
![github](https://user-images.githubusercontent.com/90299857/132443552-fdf72700-2f79-4ad5-8569-8db19f0a3134.PNG)

**Comment by tobiasK2001** [comment, marked as answer]
Dear Harry,

yes your absorption seems  to be extremely low. Check your assumptions for solubility and intestinal permeability. You can simply increase the values 10 fold and rerun the simulation to see, if there is same impact and fraction absorbed get more close to 1.

Best, Tobias

**Reply by harrythmia** [reply]
Thank you for the guidance. I'll have a look at it and see how it goes.

Yours,
Harry
